test/integration/DelayedRecovery.js

```diff
@@ -313,15 +311,15 @@ describe('Delayed Recovery', async () => {
     const mockToken = mockTokenDeployerInstance.addresses.MockToken;
 
     const organizationOwnerAddress = deployerAddress;
-    const { Organization } = Mosaic.ContractInteract;
+    const { Organization } = Package;
```

Comment:
shouldn't you instead require the organization in the head? Same in other test files.

Reply:
Done!